products/BellHybrid/apps/common/include/common/widgets/TimeSetSpinnerVertical.hpp

```diff
@@ -51,9 +51,8 @@ namespace gui
         auto setFont(const std::string &newFontName) noexcept -> void;
         auto updateFont(TextFixedSize *elem, const std::string &fontName) noexcept -> void;
 
-        [[nodiscard]] auto getTime() const noexcept -> std::time_t;
-        [[nodiscard]] auto getHour() const noexcept -> int;
-        [[nodiscard]] auto getMinute() const noexcept -> int;
+        [[nodiscard]] auto getHour() const noexcept -> std::string;
+        [[nodiscard]] auto getMinute() const noexcept -> std::string;
```

Comment:
I think we should leave getters returning ints, especially since setters take ints as arguments. Is there any particullar reason to change those getters?

Reply:
Ok, i have switched to int type to have a consistent interface in this class